src/index.ts

```diff
@@ -1,8 +1,8 @@
-const { Client } = require('discord.js');
-const express = require('express');
-const getConfig = require('./config/get-config');
-const validateConfig = require('./config/validate-config');
-const bootstrap = require('./commands/bootstrap');
+import { Client } from 'discord.js';
+import express from 'express';
```

Comment:
```suggestion
import express, { Request, Response } from 'express';
```

Then, on line 29 -

```js
    expressApp.get('/status', (_: Request, res: Response) =>
```

Reply:
Explicit types are not necessary, as the `expressApp` will infer types, so the arguments are typed without us passing anything

Reply:
Ok... got it.